File: Proyectos  c++ (2)/Formulario de baloncesto.cpp
#include<iostream>
#include<locale.h>

using namespace std;

int main(){
setlocale(LC_CTYPE, "spanish");
		int ED , ES , PS ;
		cout<< "Este programa va a dar a conocer si una persona es admitida en un equipo de baloncesto de Bogotá" <<endl;
		cout<< "Ingrese la edad: " <<endl;
		cin>> ED;
		cout<< "Ingrese la estatura: " <<endl;
		cin>> ES;
		cout<<"Ingrese el peso: ";
		cin>> PS;
	
		
				if (ED>=18)
				{
					cout<<"Cumples con la edad "<<endl ;
					if(ES>=180)
					{
						cout<<"Cumples con la estatura  "<<endl ; 
						if(PS>=80)
						{
							cout<<"Cumples con todos los requisitos, ¡Bienvenido! "<<endl;
						}
						else
						{
							cout<<"Lo sentimos, cumples con la edad, y con la estatura, pero no cumples con el peso";
						}
					}
					else
					{
						cout<<"Cumples con la edad, pero no cumples con los demas requisitos "<<endl ;
					}
				}
	    		else 
	    		{
	    			cout<<"Lo sentimos no cumples con los requisitos requeridos";
				}
	
		
		

		
		
}

File: Proyectos Python (2)/Formulario de baloncesto.py
print ("Vamos a conocer si una persona es admitida en un equipo de baloncesto en Bogotá ")
ed = int( input ("Ingrese la edad: "))
et = int ( input ("Ingrese la estatura: "))
ps = int ( input ("Ingrese el peso: "))

if ed>=18:
    print("Cumples con la edad")
    if et>180:
        print("Cumples con la estatura")
        if ps>80:
            print("cumples con todos los requisitos, ¡Bienvenido al equipo!")
        else:
            print("Lo sentimos, cumples con la estatura y la edad, pero no cumples con los demas requisitos" )
    else:
        print("Lo sentimos, cumples con la edad pero no cumples con los demas requisitos")
else:
    print("Lo sentimos pero no cumples algunos de los requisitos requeridos ")
    
